README.rst

```diff
@@ -2629,22 +2629,6 @@ Neo4j Python client called :code:`neo4j_client`:
     with neo4j_client.driver.session() as session:
         result = session.run(compilation_result.query, parameters)
 
-Amending Parsed Custom Scalar Types
```

Comment:
Do you need to update the readthedocs version of this as well?

Reply:
Nope. There is no equivalent section in readthedocs.